x-pack/plugins/uptime/public/state/api/alerts.ts

```diff
@@ -49,7 +49,7 @@ export interface NewAlertParams extends AlertTypeParams {
 }
 
 type NewMonitorStatusAlert = Omit<
```

Comment:
I think it would be good to also match our variables to the proper terms, if you have the bandwidth.